test/integration/stackstorm/controls/st2web_test.rb

```diff
@@ -85,11 +85,12 @@
   end
 
   if os.debian?
+    # on Debian/Ubuntu nginx redirects by returning a 308
     describe http('http://localhost/', enable_remote_worker: true) do
-      its('status') { should eq 301 }
+      its('status') { should eq 308 }
     end
   elsif os.redhat?
-    # on RHEL nginx redirects to https and returns 200
+    # on RHEL nginx automatically redirects to https and returns 200
```

Comment:
Is this different result caused by different `http` client behavior or it's really nginx?

Reply:
I really think it's nginx. I tried it on my local install using the 1-liner and it did the same thing... Very odd, I know.

Reply:
I was using cURL on both